app/views/admin/protocols/show.html.erb

```diff
@@ -13,17 +13,17 @@
   </div>
   <nav class="page-nav">
     <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
-      <%= link_to "Edit protocol", edit_admin_protocol_path(@protocol), class: "btn btn-sm btn-primary", id: "Edit protocol" %>
-      <%= link_to 'New protocol drug', new_admin_protocol_protocol_drug_path(@protocol), class: "btn btn-sm btn-primary", id: "New protocol drug" %>
+      <%= link_to "Edit protocol", edit_admin_protocol_path(@protocol), class: "btn btn-sm btn-outline-primary", id: "Edit protocol" %>
+      <%= link_to 'New protocol drug', new_admin_protocol_protocol_drug_path(@protocol), class: "btn btn-sm btn-success", id: "New protocol drug" %>
```

Comment:
@dburka can we use the '+' instead of 'New' to keep the green buttons consistent across pages?

Reply:
Yes @claudiovallejo , we can

Reply:
I pushed the change @dburka